src/components/sections/SchoolPage/SchoolHeader.tsx

```diff
@@ -42,21 +42,19 @@ const SchoolHeader: FC<SchoolHeaderProps> = ({
   isPublic,
   schoolName,
   district,
-  description,
+  // description,
```

Comment:
I think we can remove that. I'd rather say to avoid commenting out the code, especially not so important pieces of it, because most likely these comments will stay in the codebase for good.

Reply:
ok!